Zend/zend_compile.h

```diff
@@ -307,6 +307,9 @@ typedef struct _zend_oparray_context {
 /* Class cannot be serialized or unserialized             |     |     |     */
 #define ZEND_ACC_NOT_SERIALIZABLE        (1 << 29) /*  X  |     |     |     */
 /*                                                        |     |     |     */
+/* Readonly class                                         |     |     |     */
+#define ZEND_ACC_READONLY_CLASS          (1 << 30) /*  X  |     |     |     */
```

Comment:
It would be better to reused ZEND_ACC_READONLY.

Reply:
Yes, initially I reused it, but there were quite a few leaks (due to zend_strings I believe). maybe there was a collision with one of the constants with the same value as `ZEND_ACC_READONLY`? Then I tried to change its value to `(1 << 30)`, but there were issues again (forgot the exact problems). But as soon as I created this constant for classes, all the tests passed. (I can retry removing `ZEND_ACC_READONLY_CLASS ` a bit later, just to be sure)